express-post-json/server.js

```diff
@@ -0,0 +1,25 @@
+import express from 'express';
+
+let PORT = 8080;
+let app = express();
+let grades = {};
```

Comment:
Generally, if there's no plan to change which value a variable references, it's best to default to using `const`.